### PR TITLE
[HOPSWORKS-2356] alerting

### PR DESCRIPTION
**Pull request by ErmiasG**

### CODE REVIEW

**Comment by ErmiasG** [comment]
logicalclocks/hops-testing-ee/pull/333

File: files/default/templates/hopsworks.email.tmpl [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+                      <strong>Summary:</strong> {{ .Annotations.summary }} <br />
+                      <strong>Description:</strong> {{ .Annotations.description }} <br />
+                    {{ end }}
+                    <strong>Details:</strong><br />
+                    <ul>
+                      <li> Project: {{ .Labels.project }} </li>
+                      {{ if .Labels.job -}} <li> Job: {{ .Labels.job }} </li> {{ end }}
+                      {{ if .Labels.executionId -}} <li>  Execution id: {{ .Labels.executionId }} </li> {{ end }}
+                      {{ if .Labels.featureGroup -}} <li>  Feature Group: {{ .Labels.featureGroup }} </li> {{ end }}
+                      {{ if .Labels.featureGroupId -}} <li>  Feature Group id: {{ .Labels.featureGroupId }} </li> {{ end }}
```
Comment (SirOibaf) [inline files/default/templates/hopsworks.email.tmpl:360]: Feature group id is useless. The alert should contain feature store name/ feature group name/feature group version.

File: files/default/templates/hopsworks.email.tmpl [hunk 2]
```diff
@@ -0,0 +1,420 @@
...
```
Comment (SirOibaf) [inline files/default/templates/hopsworks.email.tmpl:397]: Feature group id is useless. The alert should contain feature store name/ feature group name/feature group version.

File: files/default/templates/hopsworks.slack.tmpl [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+   *description:* {{ .Annotations.description }}
+
+   *Details:*
+      - *Project:* {{ .Labels.project }}
+     {{- if and .Labels.job .Labels.executionId}}
+      - *Job:* {{ .Labels.job }}
+      - *Execution id:* {{ .Labels.executionId }}
+     {{- else if and .Labels.featureGroup .Labels.featureGroupId }}
+      - *Feature group:* {{ .Labels.featureGroup }}
+      - *Feature group id:* {{ .Labels.featureGroupId }}
```
Comment (SirOibaf) [inline files/default/templates/hopsworks.slack.tmpl:25]: Feature group id is useless. The alert should contain feature store name/ feature group name/feature group version.

**Review by SirOibaf — CHANGES_REQUESTED** [review]

**Review by SirOibaf — APPROVED** [review]

**Pull request merged**